kratos/solving_strategies/strategies/line_search_strategy.h

```diff
@@ -308,12 +308,12 @@ class LineSearchStrategy
         Parameters default_parameters = Parameters(R"(
         {
             "name"                       : "line_search_strategy",
-            "max_line_search_iterations" : 5,
+            "max_line_search_iterations" : 30,
```

Comment:
also remove those changes in the` line_search` please

Reply:
Done!